Setup.md

```diff
@@ -0,0 +1,67 @@
+
+## Create a GitHub.com user account
+Setup a GitHub.com user account that will be used later in this workshop. If you have an existing GitHub.com account you will be able to use it if you are comfortable using that account to create a GitHub Organization later in the workshop.
+
+1. Visit https://github.com/join and fill in the required fields to create a user account.
+2. Select "Unlimited public repositories for free" when choosing your plan.
+3. Verify your email account to ensure you account is activated.  An activated account will be **required** in the next few exercises.
+
+## Create a GitHub Personal Access Token
+The following instructions cover how to create a Github Personal Access Token that you will use within Jenkins to connect Pipelines, Multibranch Pipelines, and Github Organization Projects to your Github repositories.
+
+1. Click on [this link to automatically select the required **Personal access token settings**](https://github.com/settings/tokens/new?scopes=repo,read:user,user:email,admin:repo_hook,admin:org_hook)
+2. Click on **Generate Token**
+3. As the success message says: **Make sure to copy your new personal access token now. You won’t be able to see it again!**  
+
+## Create a GitHub Organization
+
+Create a Github organization to use for this workshop:
+
+1. On Github navigate to **Organizations**: https://github.com/settings/organizations (after logging in) 
+2. Click on **New Organization** <p><img src="img/setup/setup-github-new-org.png" width=550/>
+3. Fill in the **Organization Name**, **Billing Email**, and click on **Create Organization** - **IMPORTANT: Suggest using {your GitHub username}-devoptics as Organization Name** <p><img src="img/setup/setup-create-org.png" width=400/>
+4. On the **Invite organization members** - just click the **Finish** button.
+
+>NOTE: Even though you have to provide an email for billing, you will not be charged anything as long as you choose the free option.
+
+## Fork the Workshop Repository
+
+The workshop utilizes the **helloapp-nodejs** repository from the [CloudBees DWJW GitHub Organization](https://github.com/cloudbees-days). Fork the **helloapp-nodejs** and **helloapp-api** repositories into the GitHub Organization that you created above (if you are not sure how to fork a repository - see this [GitHub Guide on forking](https://guides.github.com/activities/forking/)):
+
+* https://github.com/cloudbees-days/helloapp-api
+* https://github.com/cloudbees-days/helloapp-nodejs
+
+<p><img src="img/setup/setup_fork_repo.png" width=700/>
+
+## Back to the Jenkins Setup
+
+You should see the following **Create a new Pipeline** screen for your Team:
+<p><img src="img/setup/setup-success.png" width=700/>
+
+You may proceed to the next set of [**labs**](./README.md#workshop-labs) - once you have successfully completed the setup.
```

Comment:
Maybe just - You may proceed to the [**labs**] - since this isn't really a lab.

Reply:
Make sense. Noted.